### PR TITLE
Descriptions of variables / Easier use with Multi-Hives

### PR DESCRIPTION
**Pull request by DirtyCajunRice**
## Description
added descriptions of variables, and allowed for easier use with multiple hives
also added coords-only.sh which runs location-generator.py and uses the verbose output to make a coords file like legacy location-generator.py did. 
(This is the easiest method until an argparse is added for coords only to the generator)

## Motivation and Context
To clear some confusion about variables and To allow people more than 1 hive

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by JZ6** [comment]
+1

File: contrib/supervisord/coords-only.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Using this until they allow location_generator.py to allow coords only mode!
```
Comment (invisiblek) [inline contrib/supervisord/coords-only.sh]: Can you make use of this?
https://github.com/PokemonGoMap/PokemonGo-Map/pull/54

File: contrib/supervisord/README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ Features:
 
 ## How to setup
 
-For instructions on how to setup and run the tool, please refer to the project [wiki](https://github.com/PokemonGoMap/PokemonGo-Map/wiki/supervisord-config-files)
+For instructions on how to setup and run the tool, please refer to the project [wiki](https://pgm.readthedocs.io/en/develop/extras/supervisord-on-linux.html)
```
Comment (invisiblek) [inline contrib/supervisord/README.md]: this should be in a separate commit (this project will squash these all into one)

Comment (DirtyCajunRice) [inline contrib/supervisord/README.md]: I thought the point was to update the relevant wiki page with its corresponding information? i can separate if thats not the case

Comment (invisiblek) [inline contrib/supervisord/README.md]: oh, i mistaked this for the README.md in the root dir, carry on =P

Comment (DirtyCajunRice) [inline contrib/supervisord/README.md]: Ill carry on when you give me your +1 so i can merge and work straight on the wiki :p

**Comment by DirtyCajunRice** [comment]
AYYY <3
